data/json/monsters.json

```diff
@@ -5040,6 +5040,36 @@
     "description":"A gargantuan protoplasmic blob, constantly reshaping, forming new pseudopods seemingly at will.  All over its body are eyes that form and disappear.  It looks at you with malice.",
     "flags":["SEES", "SMELLS", "SWIMS", "PLASTIC", "SLUDGEPROOF", "ACID_BLOOD", "ACID_PROOF", "NOHEAD", "REGENERATES_50", "ABSORBS", "REGEN_MORALE", "NOGIB"]
     },{
+    "type":"MONSTER",
+    "id":"mon_albino_penguin",
+    "name": "albino penguin",
+    "species":"NETHER",
+    "default_faction":"nether",
+    "symbol":"V",
+    "color":"white",
+    "size":"HUGE",
+    "material":"flesh",
+    "diff":11,
+    "aggression":0,
+    "morale":100,
+    "speed":50,
+    "melee_skill":7,
+    "melee_dice":4,
+    "melee_dice_sides":6,
+    "melee_cut":8,
+    "dodge":2,
+    "armor_bash":6,
+    "armor_cut":2,
+    "vision_day":5,
+	"vision_night":5,
```

Comment:
Tabs
Need to be 4 spaces instead

Any line that looks "weird" in github while looking OK in text editor has tabs.